pkg/rtc/room.go

```diff
@@ -1173,27 +1171,27 @@ func (r *Room) sendParticipantUpdates(updates []*participantUpdate) {
 
 // for protocol 2, send all active speakers
 func (r *Room) sendActiveSpeakers(speakers []*livekit.SpeakerInfo) {
-	dp := &livekit.DataPacket{
-		Kind: livekit.DataPacket_LOSSY,
-		Value: &livekit.DataPacket_Speaker{
-			Speaker: &livekit.ActiveSpeakerUpdate{
-				Speakers: speakers,
-			},
-		},
-	}
+	const kind = livekit.DataPacket_LOSSY
 
 	var dpData []byte
 	for _, p := range r.GetParticipants() {
```

Comment:
we can deprecate this function. all client SDKs supports speaker changed now.

Reply:
Done